src/btreemap.rs

```diff
@@ -2573,77 +2624,20 @@ mod test {
         };
     }
 
-    #[test]
-    #[should_panic(expected = "max_key_size must be <= 0")]
-    fn panics_if_max_key_grows() {
```

Comment:
This test is removed as it's already covered by v1_rejects_increases_in_max_key_size